test/functionalTest/cases/0997_GET_v2_entity_single_attribute_value/GET_v2_entity_single_attribute_value.test

```diff
@@ -180,65 +180,41 @@ Date: REGEX(.*)
 02. GET /v2/entities/8787GHY/attrs/brand/value
 ==============================================
 HTTP/1.1 200 OK
-Content-Length: 61
-Content-Type: application/json
+Content-Length: 13
+Content-Type: text/plain
 Date: REGEX(.*)
 
-{
-    "value": {
-        "metadata": {},
-        "type": null,
-        "value": "Mercedes Benz"
-    }
-}
+Mercedes Benz
```

Comment:
Yeah, I was also unsure about this.

Would it not be nice to have:
  Content-Type: text/number
or something similar?

"Mercedes Benz" is not very good (apart from being a bit ugly), as the users will have to strip the citation marks ...